.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 .env
-opensearch-report*
\ No newline at end of file
+opensearch-report*
+package-lock.json
```

Comment:
Earlier there was a discussion to keep yarn.lock and not package-lock.json, that's why I added here. When we run `npm install`, package-lock.json is created. Should we keep both or remove yarn.lock since we are doing npm install for setup?

Reply:
i was suggesting to keep one. if we use yarn, then we add yarn.lock to git and add yarn to instructions. If npm is recommended, then we do that for package-lock.json. either way i don't see why we need to ignore lockfile

Reply:
removed this.